### PR TITLE
Add 'AdditionalCommands' to run subcommands during bulk Rush actions

### PR DESCRIPTION
**Pull request by D4N14L**

### CODE REVIEW

**Comment by D4N14L** [comment]
Needs tests and `rush change`

File: apps/rush-lib/src/logic/taskRunner/ProjectTask.ts [hunk 1]
```diff
@@ -58,24 +63,26 @@ export class ProjectTask implements ITaskDefinition {
...
     this._rushConfiguration = options.rushConfiguration;
-    this._commandToRun = options.commandToRun;
     this.isIncrementalBuildAllowed = options.isIncrementalBuildAllowed;
     this._packageChangeAnalyzer = options.packageChangeAnalyzer;
     this._packageDepsFilename = options.packageDepsFilename;
-}
+
+    this._commandsToRun = options.commandsToRun;
+    if (this._commandsToRun.length < 1) {
+      throw Error("No commands were defined for the task.");
```
Comment (sachinjoseph) [inline apps/rush-lib/src/logic/taskRunner/ProjectTask.ts]: _commandsToRun could also be empty if --skip parameter has all parameters listed as values, right?

**Comment by sachinjoseph** [comment]
Should we clearly differentiate between commands and subcommands?

**Comment by D4N14L** [comment]
After a longer set of discussions, we've decided to expand the application of this feature making this design out-of-date

**Pull request closed without merging**